### PR TITLE
Fantasy

### PR DESCRIPTION
**Pull request by FantasyJXF**
add some files and fix some files.

keep track the original web

### CODE REVIEW

File: zh/flight_controller/mindpx.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+|1|Power|9|I2C2 (MindFLow)|
+|2|Debug (refresh bootloader)|10|USB2 (Serial 2 to USB)|
+|3|USB1 (refresh firmware)|11|UART4,5|
+|4|Reset|12|UART1 (Telemetry)|
+|5|UART3 (GPS)|13|CAN|
+|6|I2C1(external compass)|14|ADC|
+|7|TF card slot|15|Tricolor Light|
+|8|NRF/SPI(Remote Control)|16|Looper|
+
+[](../../assets/hardware/hardware-mindpx-looper.png)
```
Comment (hamishwillee) [inline zh/flight_controller/mindpx.md]: Note has preceding !

**Comment by hamishwillee** [comment]
Hi @FantasyJXF 

Unfortunately I fixed some English stuff today that clashes with this and which will affect your translations - #193 . For example, you'll see the mindxxxx docs had mis-named images, a few images missing the preceding !, and some mailto links that need to have their format changed.

Can you rebase your changes and translate the (fairly minor) changes into your docs?

**Comment by FantasyJXF** [comment]
plz check it again

**Comment by hamishwillee** [comment]
Thanks @FantasyJXF - looks great - merged.

**Pull request merged**